package.json

```diff
@@ -60,6 +60,7 @@
     "husky": "^8.0.2",
     "lint-staged": "^10.5.4",
     "prettier": "^2.7.1",
+    "ramda.clone": "^0.26.1",
```

Comment:
If you use this in the source code, it should be in the `"dependencies"` section, isn't it?

Reply:
And why not [`structuredClone`](https://caniuse.com/?search=structuredClone)

Reply:
> If you use this in the source code, it should be in the `"dependencies"` section, isn't it?

Actually all external dependencies(ex solid-js) marked as devDependencies. https://github.com/effector/inspector/pull/49/

Reply:
> And why not [`structuredClone`](https://caniuse.com/?search=structuredClone)?

Current polyfill do not implements a lot of structures and I don't know what range of browser you guys support, so I decided to pick most compatible variant and ramda.clone was most reliable.

In case you're ok with it, I'll change ramda to clone.